charts/umbrella/README.md

```diff
@@ -5,6 +5,13 @@ consisting of [Tractus-X](https://projects.eclipse.org/projects/automotive.tract
 
 The Chart aims for a completely automated setup of a fully functional network, that does not require manual setup steps.
 
+## Precondition
+In case of enabling `semantic-hub` the fuseki docker image must be built.
+Build fuseki docker image by following the below steps:
+- Download [jena-fuseki-docker-4.7.0.zip](https://repo1.maven.org/maven2/org/apache/jena/jena-fuseki-docker/4.7.0/jena-fuseki-docker-4.7.0.zip)
+- Unzip the jena-fuseki-docker-4.7.0.zip.
+- Build the docker image by running the command - `docker build --build-arg JENA_VERSION=4.7.0 -t jena-fuseki-docker:4.7.0 .`
```

Comment:
I also needed to specify ` --platform linux/amd64` on my Mac as there is no arm base image for openjdk. Also while building the image step 9 in the Dockerfile hangs for me and never gets finished.

```Dockerfile
RUN \
  JDEPS="$(jdeps --multi-release base --print-module-deps --ignore-missing-deps ${FUSEKI_JAR})"  && \
  jlink \
        --compress 2 --strip-debug --no-header-files --no-man-pages \
        --output "${JAVA_MINIMAL}" \
        --add-modules "${JDEPS},${JDEPS_EXTRA}"
```

Reply:
Added the platform option to the build command.